File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utils/Reduction.java
package org.firstinspires.ftc.teamcode.utils;

public class Reduction {
    private final double ratio;

    public Reduction(double ratio) {
        this.ratio = ratio;
    }

    public double apply(double value) {
        return value / ratio;
    }

    public double unApply(double value) {
        return value * ratio;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utils/Reduction.kt
package org.firstinspires.ftc.teamcode.utils

import android.icu.util.Measure

class Reduction(private val ratio: Double) {

    fun apply(value: Double) = value.div(ratio)
    
    fun unApply(value: Double) = value.times(ratio)

    fun uselessFun() {}
}

